### PR TITLE
bulk delete test

### PR DESCRIPTION
**Pull request by RedOrNot**

### CODE REVIEW

**Review by vishsanghishetty — DISMISSED** [review]

File: cypress/support/hostsfunctions.ts [hunk 1]
```diff
@@ -92,6 +110,15 @@ function navigateToHost(url: string, name: string, data: string) {
   cy.get(data).click();
 }
 
+function navigateToBase(host_type: string, invenotry: Inventory) {
+  if (host_type === 'inventory_host') {
+    cy.visit(`/infrastructure/inventories/inventory/${invenotry.id}/details`);
```
[the remark this answers is not in the document]
Comment (RedOrNot) [reply cypress/support/hostsfunctions.ts]: done!

File: cypress/support/hostsfunctions.ts [hunk 2]
```diff
@@ -175,7 +200,34 @@ export function createAndEditAndDeleteHost(host_type: string, inventory: Invento
...
   // delete
-  deleteHost(inventory.id, host_type, hostName);
+  deleteHost(inventory.name, host_type, hostName);
+}
+
+export function testHostBulkDelete(host_type: string, inventory: Inventory) {
+  //navigate to base screen
+  navigateToBaseView(host_type, inventory.name);
+  //create
+  const hostName1 = createAndCheckHost(host_type, inventory.name);
```
Comment (MilanPospisil) [inline cypress/support/hostsfunctions.ts:212]: Please try to use API calls using some support function, we dont want to create data manualy in tests, that are not particulary focused on test manual creation. Its for speedup the tests.

File: cypress/support/hostsfunctions.ts [hunk 3]
```diff
@@ -175,7 +200,34 @@ export function createAndEditAndDeleteHost(host_type: string, inventory: Invento
...
+  navigateToBaseView(host_type, inventory.name);
+  const hostName2 = create2ndHost(host_type, inventory.name);
```
Comment (MilanPospisil) [inline cypress/support/hostsfunctions.ts:214]: The same as above.

File: cypress/support/hostsfunctions.ts (hunk 2 above)
Comment (RedOrNot) [reply cypress/support/hostsfunctions.ts:212]: that's the main goal of this - to check user UI actions

File: cypress/support/hostsfunctions.ts (hunk 3 above)
Comment (RedOrNot) [reply cypress/support/hostsfunctions.ts:214]: this I'll change

**Comment by RedOrNot** [comment]
new PR for bulk delete since the rebase isn't working for me :
https://github.com/ansible/ansible-ui/pull/2543

**Pull request closed without merging**